app/uk/gov/hmrc/tai/viewModels/taxCodeChange/TaxCodeChangeViewModel.scala

```diff
@@ -33,12 +34,14 @@ object TaxCodeChangeViewModel extends TaxCodeDescriptor {
     TaxCodeChangeViewModel(taxCodePairs, changeDate, scottishTaxRateBands)
   }
 
-  def getTaxCodeExplanations(taxCodeRecord: TaxCodeRecord, scottishTaxRateBands: Map[String, BigDecimal])
+  def getTaxCodeExplanations(taxCodeRecord: TaxCodeRecord, scottishTaxRateBands: Map[String, BigDecimal], identifier: String)
                             (implicit messages: Messages): DescriptionListViewModel = {
 
+    val currentTaxCode = identifier == "current"
```

Comment:
can this be named with an 'is' before to clarify that it's a Boolean rather than an actual tax code?

Reply:
Agreed, it's done now